### PR TITLE
feat(dashboard): support cross-account widgets :)

### PR DESCRIPTION
**Pull request by sanderblue**
Resolves: #561 

Checklist:
- [x] Update docs

### CODE REVIEW

**Review by zlesnr — APPROVED** [review]
🦙 it.

File: website/docs/r/dashboard.html.markdown [hunk 1]
```diff
@@ -160,6 +170,10 @@ The optional filter block supports the following arguments:
   * `event_types` - (Optional) A list of event types to enable filtering for.
   * `attributes` - (Optional) A list of attributes belonging to the specified event types to enable filtering for.
 
+## Additional Examples
+
+A
```
Comment (zlesnr) [inline website/docs/r/dashboard.html.markdown:175]: Extra comment?

**Review by zlesnr — COMMENTED** [review]
This looks good to me, just one comment about the comment.

File: website/docs/r/dashboard.html.markdown (hunk 1 above)
Comment (sanderblue) [reply website/docs/r/dashboard.html.markdown:175]: Adding another example now, then this will be good to go I think.

**Review by zlesnr — APPROVED** [review]
:goat: :cloud:

**Pull request merged**